presto-main/src/main/java/io/prestosql/sql/planner/ExpressionInterpreter.java

```diff
@@ -971,7 +971,7 @@ protected Object visitBindExpression(BindExpression node, Object context)
         {
             List<Object> values = node.getValues().stream()
                     .map(value -> process(value, context))
-                    .collect(toImmutableList());
+                    .collect(Collectors.toList());
```

Comment:
static import + comment

```suggestion
                    .collect(toList()); // values are nullable
```